ComposureWatch/src/Components/Card/CardList.jsx

```diff
@@ -16,6 +16,7 @@ const CardList = (props) => {
           character={card.character}
           rank={card.rank}
           rating={card.rating}
+          cardRemoveChangeHandler={props.cardRemoveChangeHandler}
```

Comment:
you could potentially name this `onRemoveCard` or something.

Might make more sense if I saw
`<CardList onRemoveCard={removeCardHandler} />` for example.